src/tests/ftest/ior/crash.py

```diff
@@ -74,10 +89,28 @@ def test_ior_crash(self):
         if not check_system_query_status(scan_info):
             self.fail("One or more engines crashed")
 
+        # Verify container handle opened by ior is closed (by daos_agent after ior crash)
+        checks = {
+            "ci_nhandles": 1}
+        attempts = 0
+        while attempts < 5:
+            attempts += 1
+            chkres = self.container.check_container_info(**checks)
+            if chkres is True:
+                break
+            time.sleep(2)
+        self.assertTrue(chkres, "Error confirming container info nhandles")
+
```

Comment:
Maybe add a helper method for the duplicated check?

Reply:
in latest commit, a new method is added for this